src/config.ts

```diff
@@ -24,3 +25,7 @@ export class DiscordBridgeConfigAuth {
 class DiscordBridgeConfigLogging {
   public level: string;
 }
+
+class DiscordBridgeConfigRoom {
+  public defaultPublic: boolean;
```

Comment:
I'd prefer this option to be a string and we can just ask the user to specify public or invite, to make this more explicit.

Reply:
Yeah, good idea.